src/main/resources/assets/corpse/lang/en_us.json

```diff
@@ -1,21 +1,21 @@
 {
   "entity.corpse.corpse": "Corpse",
   "entity.corpse.corpse_of": "Corpse of %s",
-  "button.corpse.previous": "Previous",
+  "button.corpse.previous": "Back",
   "button.corpse.next": "Next",
   "gui.corpse.page": "%s of %s",
-  "gui.corpse.death_history.title": "Death History",
+  "gui.corpse.death_history.title": "Player Death History",
```

Comment:
It is obvious that it is the players death history.

Reply:
If you say so. One time I tried to display the death history of a specified non-player entity, and then Minecraft told me that the entity was not a player, and I was a bit surprised. Addition of word "player" here and there has no negative effects and really helps.